### DISCUSSION TITLE
Issue with connecting to Lavalink 4.0.0.

### DISCUSSION THREAD

**Discussion by ScriptVortexDE** [lavalink-devs/Lavalink#994] [Q&A] [answered]
## Description
It's like this, you install Lavalink normally via the installer of pterodactyl there the latest version is taken (currently 4.0.0), if you try to connect now you get a /get error for the path /

But if I now connect to my old Lavalink server, (currently 3.7.8) you can connect without any problems

Video for the error picture 

https://auroa.link/view/0ae05839-d7ea-4f99-97d8-104775513bb9.mp4

## To Reproduce
1. Install Lavalink Fresh Pterodactyl
2. Set password
3. Connect with service ( Multibot in my fale )
4. See error http://45.92.216.179:43592/nixepahoto.yaml

       .   _                  _ _       _    __ _ _
      /\\ | | __ ___   ____ _| (_)_ __ | | __\ \ \ \
     ( ( )| |/ _` \ \ / / _` | | | '_ \| |/ / \ \ \ \
      \\/ | | (_| |\ V / (_| | | | | | |   <   ) ) ) )
       '  |_|\__,_| \_/ \__,_|_|_|_| |_|_|\_\ / / / /
    =========================================/_/_/_/

        Version:        4.0.0
        Build time:     03.12.2023 14:46:34 UTC
        Branch          HEAD
        Commit:         acc6069
        Commit time:    03.12.2023 14:45:17 UTC
        JVM:            17.0.9
        Lavaplayer      2.0.4
        
        
With the version 3.7.8 its working fine 

That can you see in the hastebin log file 
http://45.92.216.179:43592/nixepahoto.yaml

**Comment by duncte123** [comment, marked as answer]
Lavalink v4 is not compatible with lavalink v3 (see [semver](https://semver.org/) about how the version is structured). you need to use an updated client https://lavalink.dev/clients.html
